### DISCUSSION TITLE
How to listen to connection status?

### DISCUSSION THREAD

**Discussion by adlanarifzr** [mikro-orm/nestjs#174] [General]
I am looking for a way to listen to database connection status like "Connected" or "Disconnected" or "Reconnecting" or others. How am I supposed to do this? I did not find any way in the documentation.

**Comment by B4nan** [comment]
This is all handled by knex, also there is a pool of connections, its not a single connection, so listening to such events does not make much sense to me (if they even exist - I would say there will be hooks for creating and destroying connections from pool instead of that).
